libs/blocks/chart/list.js

```diff
@@ -14,15 +23,15 @@ export function listChartData(json) {
     json[tableKey].data?.forEach((column) => {
```

Comment:
Could you could also use listToLowerCase here?
```suggestion
    listToLowerCase(json[tableKey].data)?.forEach((column) => {
```

Reply:
I'm wary of updating that here because it's essentially looping over the data twice with the map/reduce and then the forEach